openasip/Makefile.am

```diff
@@ -6,11 +6,7 @@ SUBDIRS += bclib
 SUBDIRS += newlib-1.17.0/tce-llvm 
 SUBDIRS += newlib-1.17.0/tcele-llvm 
 SUBDIRS += newlib-1.17.0/tcele64-llvm
-
-if COMPILE_RISCV_NEWLIB
-MAKE_RISCV_NEWLIB=true
```

Comment:
Does this require the RISC-V target built in to the LLVM to succeed? Does the OpenASIP LLVM installer script enable it by default?

Reply:
It indeed requires the RISC-V target to be built but this is done automatically in the LLVM installer script. Newlib can be successfully built without the RISC-V GNU Toolchain however.